Coding-Interview-Codes/Readme.md

```diff
@@ -78,3 +78,37 @@ Input : arr[] = {20, 10, 20, 4, 100}
 Output : 100
 
 The solution is to initialize max as first element, then traverse the given array from second element till end. For every traversed element, compare it with max, if it is greater than max, then update max.
+
+# 4. Fibonacci series(fibonacci.cpp)
+
+## Problem Statement :
+
+Given a number n, the task is to write a program in C++ to print the n-terms of Fibonacci Series using a Class template
+The Fibonacci numbers are the numbers in the following integer sequence.
+
+0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, .....
+
+### Examples:
+
+Input: n = 2
+Output: 0, 1
+
+Input: n = 9
+Output: 0, 1, 1, 2, 3, 5, 8, 13, 21
+
+### Explanation:
+
+1. Create a class for the Fibonacci Series
+
+2. Take the first two terms of the series as public members a and b with values 0 and 1, respectively.
+
+3. Create a generate() method in this class to generate the Fibonacci Series.
+
+4. Create an object of this class and call the generate() method of this class using that object.
+
+5. The Fibonacci Series will get printed.
+
+## [Different Patterns Codes](https://github.com/Shubhamnakawe/BackToBasics-Hacktoberfest/tree/main/Coding-Interview-Codes/Patterns)
```

Comment:
You will have to change this link to our repository not yours. It won't be good if someone clicks on a link and it takes them somewhere entirely different.

Reply:
You just need to change the username to `MahaWiki` that's all